q_pup_config/config.json

```diff
@@ -0,0 +1,211 @@
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "upper_leg": [
+                    0.0906,
+                    -0.0815,
+                    -0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "lower_leg": [
+                    0.0,
+                    0.0,
+                    -0.4,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "foot": [
+                    0.0,
+                    0.0,
+                    -0.35,
+                    0.0,
+                    0.0,
+                    0.0
+                ]
+            },
+            "left_hind": {
+                "hip": [
+                    -0.3149,
+                    0.2384,
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "upper_leg": [
+                    -0.0906,
+                    0.0815,
+                    -0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "lower_leg": [
+                    0.0,
+                    0.0,
+                    -0.4,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "foot": [
+                    0.0,
+                    0.0,
+                    -0.35,
+                    0.0,
+                    0.0,
+                    0.0
+                ]
+            },
+            "right_hind": {
+                "hip": [
+                    -0.3149,
+                    -0.2384,
+                    0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "upper_leg": [
+                    -0.0906,
+                    -0.0815,
+                    -0.0,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "lower_leg": [
+                    0.0,
+                    0.0,
+                    -0.4,
+                    0.0,
+                    0.0,
+                    0.0
+                ],
+                "foot": [
+                    0.0,
+                    0.0,
+                    -0.35,
+                    0.0,
+                    0.0,
+                    0.0
+                ]
+            }
+        },
+        "gait": {
+            "knee_orientation": ">>",
+            "odom_scaler": 1.0,
+            "max_linear_vel_x": 0.25,
+            "max_linear_vel_y": 0.2,
+            "max_angular_vel_z": 1.0,
+            "stance_duration": 0.5,
+            "com_x_translation": 0.0,
+            "swing_height": 0.08,
+            "stance_depth": 0.0,
+            "nominal_height": 0.5,
+            "pantograph_leg": "false"
+        }
```

Comment:
`max_linear_vel_x`: default is 0.5 (m/s), halved it based on napkin math & max speed
`stance_duration`: this is how long a foot stays on the ground, doubled it from default 0.25 (s), thought it may be better to leave feet on the ground longer
`swing_heigh`: this means the foot lifts 8cm off the ground, default is 4cm
`nominal_height`: this is basically how tall the bot stands, i set it to 0.5m

everything else is default